vald/evm/evm.go

```diff
@@ -253,6 +274,44 @@ func (mgr Mgr) ProcessTransferKeyConfirmation(event *types.ConfirmKeyTransferSta
 	return err
 }
 
+func (mgr Mgr) isStuckFileCoinKeyRotation(event *types.ConfirmKeyTransferStarted) bool {
+	if mgr.chainID != "axelar-testnet-lisbon-3" {
+		return false
+	}
+	if event.PollID.String() != "1342934" {
```

Comment:
Every confirmation will use a new poll ID counter. We can't retry voting on the same poll id, so this check should be removed